websocket/src/websocket.cpp

```diff
@@ -380,7 +436,7 @@ static dmExtension::Result WebsocketAppInitialize(dmExtension::AppParams* params
     return dmExtension::RESULT_OK;
 }
 
-static dmExtension::Result WebsocketInitialize(dmExtension::Params* params)
+static dmExtension::Result Initialize(dmExtension::Params* params)
```

Comment:
They're already in the `dmWebSocket` namespace, so it feels redundant to name it "dmWebSocket::WebsocketInitialize`